Fix bug 283740
Backport of:
From a56102700eb77cc6563b99b68acdc699a6716e73 Mon Sep 17 00:00:00 2001
From: Tao Fei <Filia.Tao@gmail.com>
Date: Fri, 03 Jul 2009 11:20:56 +0000
Subject: fix name confilct bug in beaglePrefs.js (for firefox 3.5)

remove some debug msg (break in firefox 3.5)
update zh-cn locale for beagle search history sidebar
update max firfox version to 3.5.*
update version number to 1.1.2

From upstream git.
Link: http://git.gnome.org/cgit/beagle/commit/?id=a56102700eb77cc6563b99b68acdc699a6716e73

diff -NrU5 beagle-0.3.9.orig/firefox-extension/chrome/content/beaglePrefs.js beagle-0.3.9/firefox-extension/chrome/content/beaglePrefs.js
--- beagle-0.3.9.orig/firefox-extension/chrome/content/beaglePrefs.js	2009-09-05 12:28:10.000000000 +0200
+++ beagle-0.3.9/firefox-extension/chrome/content/beaglePrefs.js	2009-09-05 12:29:07.000000000 +0200
@@ -2,11 +2,11 @@
  * Beagle Extension: Index webpages you visit using the Beagle Indexing Engine.
  * An Extension for the Firefox Browser.
  */
 
 // Initiate a new preference instance.
-var gPrefService = Components.classes['@mozilla.org/preferences-service;1'].getService(Components.interfaces.nsIPrefBranch);
+_prefService = Components.classes['@mozilla.org/preferences-service;1'].getService(Components.interfaces.nsIPrefBranch);
 
 var beaglePref = {
    
     //some constant
     RULE_INCLUDE : 1,
@@ -29,18 +29,18 @@
 
    
     //functions used to get/set pref
     func_factory:{
         'get':{
-            'bool': Function.bind(gPrefService.getBoolPref,gPrefService),
-            'int': Function.bind(gPrefService.getIntPref,gPrefService),
-            'string' : Function.bind(gPrefService.getCharPref,gPrefService)
+            'bool': Function.bind(_prefService.getBoolPref,_prefService),
+            'int': Function.bind(_prefService.getIntPref,_prefService),
+            'string' : Function.bind(_prefService.getCharPref,_prefService)
         },
         'set':{
-            'bool': Function.bind(gPrefService.setBoolPref,gPrefService),
-            'int' : Function.bind(gPrefService.setIntPref,gPrefService),
-            'string' : Function.bind(gPrefService.setCharPref,gPrefService)
+            'bool': Function.bind(_prefService.setBoolPref,_prefService),
+            'int' : Function.bind(_prefService.setIntPref,_prefService),
+            'string' : Function.bind(_prefService.setCharPref,_prefService)
         }
     },
 
     prefObject : {},
     
@@ -318,8 +318,8 @@
         }
         catch(ex){
             log("first run import error");
             log(ex);
         }
-    },
+    }
 }
 
diff -NrU5 beagle-0.3.9.orig/firefox-extension/chrome/content/jslib/io/dir.js beagle-0.3.9/firefox-extension/chrome/content/jslib/io/dir.js
--- beagle-0.3.9.orig/firefox-extension/chrome/content/jslib/io/dir.js	2009-09-05 12:28:10.000000000 +0200
+++ beagle-0.3.9/firefox-extension/chrome/content/jslib/io/dir.js	2009-09-05 12:29:07.000000000 +0200
@@ -219,11 +219,11 @@
     "   readDir(aDirPath);\n";
 
   return help;
 });
 
-jslibDebug('*** load: '+JS_DIR_FILE+' OK');
+//jslibDebug('*** load: '+JS_DIR_FILE+' OK');
 
 } else {
     dump("JSLIB library not loaded:\n"                                  +
          " \tTo load use: chrome://jslib/content/jslib.js\n"            +
          " \tThen: include(jslib_dir);\n\n");
diff -NrU5 beagle-0.3.9.orig/firefox-extension/chrome/content/jslib/io/dirUtils.js beagle-0.3.9/firefox-extension/chrome/content/jslib/io/dirUtils.js
--- beagle-0.3.9.orig/firefox-extension/chrome/content/jslib/io/dirUtils.js	2009-09-05 12:28:10.000000000 +0200
+++ beagle-0.3.9/firefox-extension/chrome/content/jslib/io/dirUtils.js	2009-09-05 12:29:07.000000000 +0200
@@ -171,11 +171,11 @@
   return help;
 }
 
 }; //END CLASS 
 
-jslibDebug('*** load: '+JS_DIRUTILS_FILE+' OK');
+//jslibDebug('*** load: '+JS_DIRUTILS_FILE+' OK');
 
 } // END BLOCK JS_LIB_LOADED CHECK
 
 else {
     dump("JSLIB library not loaded:\n"                                  +
diff -NrU5 beagle-0.3.9.orig/firefox-extension/chrome/content/jslib/io/file.js beagle-0.3.9/firefox-extension/chrome/content/jslib/io/file.js
--- beagle-0.3.9.orig/firefox-extension/chrome/content/jslib/io/file.js	2009-09-05 12:28:10.000000000 +0200
+++ beagle-0.3.9/firefox-extension/chrome/content/jslib/io/file.js	2009-09-05 12:29:07.000000000 +0200
@@ -756,11 +756,11 @@
     "   help;\n";
 
   return help;
 })
 
-jslibDebug('*** load: '+JS_FILE_FILE+' OK');
+//jslibDebug('*** load: '+JS_FILE_FILE+' OK');
 
 } // END BLOCK JS_LIB_LOADED CHECK
 
 // If jslib base library is not loaded, dump this error.
 else
diff -NrU5 beagle-0.3.9.orig/firefox-extension/chrome/content/jslib/io/fileUtils.js beagle-0.3.9/firefox-extension/chrome/content/jslib/io/fileUtils.js
--- beagle-0.3.9.orig/firefox-extension/chrome/content/jslib/io/fileUtils.js	2009-09-05 12:28:10.000000000 +0200
+++ beagle-0.3.9/firefox-extension/chrome/content/jslib/io/fileUtils.js	2009-09-05 12:29:07.000000000 +0200
@@ -582,11 +582,11 @@
     return help;
   }
   
 };
   
-jslibDebug('*** load: '+JS_FILEUTILS_FILE+' OK');
+//jslibDebug('*** load: '+JS_FILEUTILS_FILE+' OK');
   
 } // END BLOCK JS_LIB_LOADED CHECK
   
 // If jslib base library is not loaded, dump this error.
 else
diff -NrU5 beagle-0.3.9.orig/firefox-extension/chrome/locale/zh-CN/beagle.dtd beagle-0.3.9/firefox-extension/chrome/locale/zh-CN/beagle.dtd
--- beagle-0.3.9.orig/firefox-extension/chrome/locale/zh-CN/beagle.dtd	2009-09-05 12:28:10.000000000 +0200
+++ beagle-0.3.9/firefox-extension/chrome/locale/zh-CN/beagle.dtd	2009-09-05 12:29:07.000000000 +0200
@@ -29,11 +29,13 @@
 <!ENTITY beagle.run.never.index.label               "从不索引这个站点">
 <!ENTITY beagle.run.preferences.label               "选项">
 <!ENTITY beagle.run.context.menu.label              "Beagle">
 <!ENTITY beagle.run.index.link.label                "索引链接">
 <!ENTITY beagle.run.index.image.label               "索引图片">
-
 <!ENTITY beagle.index.link.start.label              "开始">
 <!ENTITY beagle.index.link.stop.label               "停止">
 
 <!ENTITY beagle.bookmark.index.modified.label       "索引更改过的书签">
 
+<!ENTITY beagle.sidebar.title "Beagle 搜索历史侧边栏">
+<!ENTITY beagle.sidebar.commandkey "B">
+<!ENTITY beagle.sidebar.modifierskey "shift 键">
diff -NrU5 beagle-0.3.9.orig/firefox-extension/install.rdf.in beagle-0.3.9/firefox-extension/install.rdf.in
--- beagle-0.3.9.orig/firefox-extension/install.rdf.in	2009-09-05 12:28:10.000000000 +0200
+++ beagle-0.3.9/firefox-extension/install.rdf.in	2009-09-05 12:29:50.000000000 +0200
@@ -21,11 +21,11 @@
 
     <em:targetApplication>
       <Description>
         <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
         <em:minVersion>0.8</em:minVersion>
-        <em:maxVersion>3.0.*</em:maxVersion>
+        <em:maxVersion>3.6.*</em:maxVersion>
       </Description>
     </em:targetApplication>
 
   </Description>
 
diff -NrU5 beagle-0.3.9.orig/firefox-extension/Makefile.am beagle-0.3.9/firefox-extension/Makefile.am
--- beagle-0.3.9.orig/firefox-extension/Makefile.am	2009-09-05 12:28:10.000000000 +0200
+++ beagle-0.3.9/firefox-extension/Makefile.am	2009-09-05 12:29:07.000000000 +0200
@@ -1,8 +1,8 @@
 
 PROJECT = beagle
-VERSION = 1.1.0
+VERSION = 1.1.2
 
 INSTALL_RDF = install.rdf
 INSTALL_RDF_IN = $(INSTALL_RDF).in
 VERSION_SED = sed -e "s|\@VERSION\@|$(VERSION)|g"
 
